src/auth/test/auth.controller.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { CreateUserDto } from './../dto/create_user.dto';
+import { AuthService } from './../auth.service';
+import { Test, TestingModule } from '@nestjs/testing';
+import { AuthController } from './../auth.controller';
+import { UserType } from '../user_type.enum';
+
+describe('AuthController', () => {
+  let authController: AuthController;
+  const createUserDto: CreateUserDto = {
+    email: 'bbrizolara7@gmail.com',
+    name: 'Bruno',
+    password: 'Bruno123!',
+    type: UserType.ADMIN
+  };
+  const mockAuthService = {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    signUp: jest.fn(_createUserDto => {
+      return {
+        token: 'usertoken'
+      };
+    })
+  };
+
+  beforeEach(async () => {
```

Comment:
You don't need to initialize the module before each test case. You should use `beforeAll` WDYT?